.vscode/launch.json

```diff
@@ -1,77 +1,58 @@
 {
-    // Use IntelliSense to learn about possible attributes.
-    // Hover to view descriptions of existing attributes.
-    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-    "version": "0.2.0",
-    "configurations": [
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
Please remove these files and gitignore them.

Reply:
"Keith has done a lot and taken several steps without really checking in and explaining that was the way that he was going to do things. " - this was marked as a draft for the purpose of discussion, so honestly thank you for the for the feedback!

"creating an entire new class to have as a message instead of attempting to deal with the information as it comes in"
- Not sure this is more complex, other than it actually catches and parses messages by...
- Single entry-point for all messages seemed like the most efficient approach
- commands are defined in a single commands array. No need for switch statements to add new commands. Add the command and a handler.
- Given single responsibility, each handler should be in a separate class within a final PR.